pyproject.toml

```diff
@@ -0,0 +1,22 @@
+[build-system]
+requires = ["scikit-build-core"]
+build-backend = "scikit_build_core.build"
+
+[project]
+name = "tt-umd"
+version = "1.0.0"
+authors = [
+    { name = "Tenstorrent", email = "info@tenstorrent.com" }
+]
+description = "User Mode Driver for tenstorrent"
+readme = "README.md"
+license = { text = "Apache-2.0" }
+classifiers = [
```

Comment:
you may want to chase down expanding this:

https://github.com/tenstorrent/tt-smi/blob/3add8b166d1e64b04a566a2847df50199b97bc21/pyproject.toml#L14

and


https://pypi.org/classifiers/